kujira/app/mirage/config.js

```diff
@@ -8,7 +52,7 @@ export default function() {
     Note: these only affect routes defined *after* them!
   */
   // this.urlPrefix = '';    // make this `http://localhost:8080`, for example, if your API is on a different server
-  this.namespace = '/kujira/api/v1';    // make this `api`, for example, if your API is namespaced
+  //this.namespace = '/kujira/api/v1';    // make this `api`, for example, if your API is namespaced
```

Comment:
Lines of code before this.namespace are not affected by it.
And second thing is line no. 5: this.namespace = config.APP.NAMESPACE;
So this is actually the same, but better in my opinion (it gets namespace from config).